### PR TITLE
Miscellaneous version updates for .NET 6 release

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to #27139

### CODE REVIEW

File: docs/core/tools/dotnet-new-sdk-templates.md [hunk 1]
```diff
@@ -169,6 +170,7 @@ Each template may have additional options available. The core templates have the
```
Comment (gewarren) [inline docs/core/tools/dotnet-new-sdk-templates.md:170]: Is the default version of net5.0 on line 115 still correct?

Comment (tdykstra) [reply docs/core/tools/dotnet-new-sdk-templates.md:170]: @vlada-shubina What is the default value for --framework for the wpf templates in .NET 6 SDK?

File: docs/standard/frameworks.md [hunk 2]
```diff
@@ -16,15 +16,16 @@ An app or library can also target a specific .NET implementation to gain access
...
 
-The following table defines the most common target frameworks, how they're referenced, and which version of [.NET Standard](net-standard.md) they implement. These target framework versions are the latest stable versions. Pre-release versions aren't shown. A target framework moniker (TFM) is a standardized token format for specifying the target framework of a .NET app or library.
+The following table defines the most common target frameworks, how they're referenced, and which version of [.NET Standard](net-standard.md) they implement. These target framework versions are the latest stable versions. Prerelease versions aren't shown. A target framework moniker (TFM) is a standardized token format for specifying the target framework of a .NET app or library.
 
 | Target framework      | Latest <br/> stable version | Target framework moniker (TFM) | Implemented <br/> .NET Standard version |
 | :-: | :-: | :-: | :-: |
-| .NET 5                | 5.0                         | net5.0                         | N/A                                     |
+| .NET 5                | 6                           | net6.0                         | N/A                                     |
```
Comment (tdykstra) [inline docs/standard/frameworks.md:27]:
```suggestion
| .NET 6                | 6                           | net6.0                         | N/A                                     |
```

**Review by tdykstra — APPROVED** [review]
LGTM aside from one typo and one unanswered question

File: docs/standard/frameworks.md (hunk 2 above)
Comment (gewarren) [reply docs/standard/frameworks.md]: Good catch!

File: docs/core/tools/dotnet-new-sdk-templates.md (hunk 1 above)
Comment (vlada-shubina) [reply docs/core/tools/dotnet-new-sdk-templates.md:170]: The default value for wpf template is net6.0

**Comment by gewarren** [comment]
Thanks Tom!

**Pull request merged**